README.md

```diff
@@ -109,6 +119,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries related to handling date and time.*
 
+* [JDateTime](http://jodd.org/doc/jdatetime.html) - Developer-friendly, astronomically precise date/time class based on Julian dates.
```

Comment:
+ only few classes in the package
+ it also gives you [Julian date number](http://en.wikipedia.org/wiki/Julian_day) (JDN), so it is very easy to calc distance between two time points - just using add/sub. Distance between two JDN is _always_ 1.
+ Uses astronomical algorithm; hence very stable and precise

btw, it exist even before JodaTime :)